### ISSUE TITLE
DISABLE_IPINFO="true" not working properly

### ISSUE THREAD

**Issue by jospanier** [librespeed/speedtest#415]
## Description
When using docker image, setting environment `DISABLE_IPINFO="true"` is not working properly and one still gets "`GET /backend/getIP.php?isp=true&....`"

## Steps to reproduce
`sudo docker run --rm -it -e MODE="standalone" -e DISABLE_IPINFO="true" -p 80:80 adolfintel/speedtest`

## Expected behaviour
`GET /backend/getIP.php?isp=false&...."`

## Log
```
sudo docker run --rm -it -e MODE="standalone" -e DISABLE_IPINFO="true" -p 80:80 adolfintel/speedtest
+ rm -rf '/var/www/html/*'
+ cp /speedtest/speedtest.js /speedtest/speedtest_worker.js /var/www/html/
+ cp /speedtest/favicon.ico /var/www/html/
+ '[' standalone == standalone ']'
+ cp -r /speedtest/backend/ /var/www/html/backend
+ '[' '!' -z '' ']'
+ '[' standalone == backend ']'
+ '[' standalone == frontend ']'
+ '[' standalone == standalone ']'
+ cp /speedtest/standalone.php /var/www/html/index.php
+ [[ false == \t\r\u\e ]]
+ chown -R www-data /var/www/html/backend /var/www/html/favicon.ico /var/www/html/index.php /var/www/html/speedtest.js /var/www/html/speedtest_worker.js
+ '[' 80 '!=' 80 ']'
+ echo 'Done, Starting APACHE'
Done, Starting APACHE
+ apache2-foreground
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 23 15:55:41.517808 2021] [mpm_prefork:notice] [pid 13] AH00163: Apache/2.4.38 (Debian) PHP/7.4.15 configured -- resuming normal operations
[Tue Mar 23 15:55:41.517918 2021] [core:notice] [pid 13] AH00094: Command line: 'apache2 -D FOREGROUND'
80.226.51.78 - - [23/Mar/2021:15:55:55 +0000] "GET /speedtest_worker.js?r=0.6645595162114482 HTTP/1.1" 200 8115 "http://10.2.130.20/" "Mozilla/5.0 (Linux; Android 6.0.1; D6603 Build/23.5.A.1.291) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.89 Mobile Safari/537.36"

80.226.51.78 - - [23/Mar/2021:15:55:55 +0000] "GET /backend/getIP.php?isp=true&distance=km&r=0.5460040577029177 HTTP/1.1" 200 435 "http://10.2.130.20/speedtest_worker.js?r=0.6645595162114482" "Mozilla/5.0 (Linux; Android 6.0.1; D6603 Build/23.5.A.1.291) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.89 Mobile Safari/537.36"

80.226.51.78 - - [23/Mar/2021:15:56:56 +0000] "GET /backend/empty.php?r=0.7374630873573631 HTTP/1.1" 200 375 "http://10.2.130.20/speedtest_worker.js?r=0.6645595162114482" "Mozilla/5.0 (Linux; Android 6.0.1; D6603 Build/23.5.A.1.291) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.89 Mobile Safari/537.36"
80.226.51.78 - - [23/Mar/2021:15:56:56 +0000] "GET /backend/empty.php?r=0.21178751115534888 HTTP/1.1" 200 375 "http://10.2.130.20/speedtest_worker.js?r=0.6645595162114482" "Mozilla/5.0 (Linux; Android 6.0.1; D6603 Build/23.5.A.1.291) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.89 Mobile Safari/537.36"
```

**Comment by jospanier** [comment]
Thinking about it, the issue may be in `standalone.php` ->`s.setParameter("getIp_ispInfo","false");` where `getIp_ispInfo` gets set to the _string_ `"false"`, which later evaluates to true because it is not an empty string

**Comment by adolfintel** [comment]
You're right, I'll fix this ASAP

**Comment by adolfintel** [comment]
Fixed in 5.2.4

**Comment by jospanier** [comment]
Many thanks.
